deployment/src/main/java/io/quarkiverse/openapi/generator/deployment/codegen/OpenApiGeneratorCodeGenBase.java

```diff
@@ -70,6 +72,14 @@ public boolean shouldRun(Path sourceDir, Config config) {
         return inputBaseDir != null || Files.isDirectory(sourceDir);
     }
 
+    protected boolean isRestEasyReactive(CodeGenContext context) {
+        List<String> appDependencies = context.applicationModel().getDependencies().stream()
+                .map(ResolvedDependency::getArtifactId)
+                .collect(Collectors.toList());
+
+        return RestEasyImplementationVerifier.get().isRestEasyReactive(appDependencies);
+    }
+
```

Comment:
I changed it to use `ExtensionCapabilities`. But anyway, I'm not sure if it makes any difference to your question. Why couldn't it work with Gradle?

Reply:
I'm just wondering why ``ResolvedDependency `` comes from ``io.quarkus.**maven**.dependency`` and would that work if an app would use ``Gradle`` to resolve the ``rest-client-reactive`` Quarkus extension ... 

Anyway, since you moved to``ExtensionCapabilities`` you'll check capabilities instead of resolved artifacts which should work in any case ..

Reply:
Got it ;)
Thanks!